lib/configuration.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+# shellcheck source-path=SCRIPTDIR
+source "$(dirname "${BASH_SOURCE[0]}")/log.sh"
+
+# Return this plugin's configuration as a JSON string.
+#
+# Assumes that there is only a single instance of this plugin
+# configured for the current job (which is a valid assumption at this
+# point).
+plugin_configuration() {
```

Comment:
(maybe add a comment like 'see an example input/output in configuration_test.sh` for these jqs)